### PR TITLE
Update TTS guide related to completed callback

### PR DESCRIPTION
**Pull request by syeonholic**
### Change Description ###

- Update TTS guide related to completed callback.
-

### CODE REVIEW

**Comment by journey2w** [comment]
Please checkout the webpage https://docs1.stg.tizen.org/staging/1917

**Review by stom-hwang — APPROVED** [review]
LGTM 👍 
And I suggest adding this description into csharp documentation.

**Comment by journey2w** [comment]
Please checkout the webpage https://docs1.stg.tizen.org/staging/1917

File: docs/application/dotnet/guides/uix/tts.md [hunk 1]
```diff
@@ -243,7 +243,24 @@ Event handlers can be set for the following events of the [Tizen.Uix.Tts.TtsClie
 
 -   Utterance started or completed
 
-    If you add text in TTS, that text is handled as an utterance and it obtains its own ID. After you request the TTS process to start, the text is synthesized by an engine and played. To get a notification when an utterance is started or completed, register event handlers for the `UtteranceStarted` and `UtteranceCompleted` events, respectively:
+    If you add text in TTS, that text is handled as an utterance and it obtains its own ID. After you request the TTS process to start, the text is synthesized by an engine and played. Each event handler is invoked in the following cases:
+    
+     |     Event handler    |               Invoked when                | TTS state |
+     | :------------------: | :---------------------------------------: | :-------: |
+     |  `UtteranceStarted`  | playing the synthesized audio is started  | `Ready` > `Playing` |
+     | `UtteranceCompleted` | playing the synthesized audio is finished | `Playing` (The state is NOT changed until `Stop()` is called.) |
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/uix/tts.md:251]:
```suggestion
     |  `UtteranceStarted`  | Playing the synthesized audio is started  | `Ready` > `Playing` |
     | `UtteranceCompleted` | Playing the synthesized audio is finished | `Playing` (The state is NOT changed until `Stop()` is called.) |
```

File: docs/application/dotnet/guides/uix/tts.md [hunk 2]
```diff
@@ -243,7 +243,24 @@ Event handlers can be set for the following events of the [Tizen.Uix.Tts.TtsClie
...
+
+    > [!NOTE]
+    > `UtteranceCompleted` is NOT invoked when
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/uix/tts.md:254]:
```suggestion
    > `UtteranceCompleted` is NOT invoked when the following occurs:
```

File: docs/application/dotnet/guides/uix/tts.md [hunk 3]
```diff
@@ -243,7 +243,24 @@ Event handlers can be set for the following events of the [Tizen.Uix.Tts.TtsClie
...
+    >
+    > (1) your application calls `Stop()`.
+    >
+    > (2) playing the synthesized audio is stopped by other application.
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/uix/tts.md:258]:
```suggestion
    > (1) Your application calls `Stop()`.
    >
    > (2) Playing the synthesized audio is stopped by another application.
```

File: docs/application/native/guides/text-input/tts.md [hunk 4]
```diff
@@ -262,7 +262,24 @@ To set and unset callbacks, follow these steps:
 
    - Utterance started or completed
 
-     If you add text in TTS, that text is handled as an utterance and it obtains its own ID. After you request the TTS process to start, the text is synthesized by an engine and played out. To get a notification when an utterance is started or completed, set the respective callbacks:
+     If you add text in TTS, that text is handled as an utterance and it obtains its own ID. After you request the TTS process to start, the text is synthesized by an engine and played out. Each callback is invoked in the following cases:
+
+     |           Callback           |               Invoked when                | TTS state |
+     | :--------------------------: | :---------------------------------------: | :-------: |
+     |  Utterance started callback  | playing the synthesized audio is started  | `TTS_STATE_READY` > `TTS_STATE_PLAYING` |
+     | Utterance completed callback | playing the synthesized audio is finished | `TTS_STATE_PLAYING` (The state is NOT changed until `tts_stop()` is called.) |
```
Comment (safir-srbd) [inline docs/application/native/guides/text-input/tts.md:270]:
```suggestion
     |  Utterance started callback  | Playing the synthesized audio is started  | `TTS_STATE_READY` > `TTS_STATE_PLAYING` |
     | Utterance completed callback | Playing the synthesized audio is finished | `TTS_STATE_PLAYING` (The state is NOT changed until `tts_stop()` is called.) |
```

File: docs/application/native/guides/text-input/tts.md [hunk 5]
```diff
@@ -262,7 +262,24 @@ To set and unset callbacks, follow these steps:
...
+
+        > [!NOTE]
+        > Utterance completed callback is NOT invoked when
```
Comment (safir-srbd) [inline docs/application/native/guides/text-input/tts.md:273]:
```suggestion
        > Utterance completed callback is NOT invoked when the following occurs:
```

File: docs/application/native/guides/text-input/tts.md [hunk 6]
```diff
@@ -262,7 +262,24 @@ To set and unset callbacks, follow these steps:
...
+        >
+        > (1) your application calls `tts_stop()`.
+        >
+        > (2) playing the synthesized audio is stopped by other application.
```
Comment (safir-srbd) [inline docs/application/native/guides/text-input/tts.md:277]:
```suggestion
        > (1) Your application calls `tts_stop()`.
        >
        > (2) Playing the synthesized audio is stopped by another application.
```

**Review by safir-srbd — COMMENTED** [review]
@syeonholic Hello. Hope you are doing well.

I have gone through the changes you have made and found a few suggestions for you to implement. Please check them.

**Pull request merged**